styles/components/button.scss

```diff
@@ -131,3 +187,33 @@
   //   border: 1px solid $gray;
   // }
 }
+
+.btn-group {
+  button,.btn + .btn{
```

Comment:
Let's not use `button` since our buttons are not specific for the `button` class, right?